src/components/FlexibleForm/ControlDecorators/withReferenceValues/withReferenceValues.js

```diff
@@ -119,7 +119,7 @@ export const withReferenceValues = memo(props => {
               id={id}
               label={localized}
               dataOptions={dataOptions}
-              optionValue="id"
+              optionValue="name"
```

Comment:
Why did you changed this to "name"?

Reply:
Because in the `OptionsList` component I changed function `onClick()`, now it passes` option?.[optionValue] || option?.[optionLabel]` to the `onSelect()` method. I did this because in the `withDatePicker` decorator we don't put options values to the input field as it is (moreover values in this decorator are rendered not as a string, but as an object), but we check the option name.
So due to changes in the `OptionsList` component, I had to change `optionValue` here.
It works well, tested it.